PermanentOrg/web-app
# Per 9660 shares from invitation

This PR adds the new view when the user without an account receives an invitation to view someone else's file.

Steps to test:
- Log in into an already existing account and share an item with an an email that is not tied to any Permanent accounts.
- Access the link from the email received on the non-existing user.
- Create an account
- You should see the new view when starting the onboarding process
